cobc/pplex.l

```diff
@@ -1629,10 +1633,21 @@ ppcopy (const char *name, const char *lib, struct cb_replace_list *replace_list)
 		}
 		/* otherwise fall-trough to error handling */
 	} else {
-		/* ensure to have errno from name as specified, not from another file */
-		(void)access (plexbuff1, R_OK);
-		/* pass file error as we have no more places to check */
-		cb_error ("%s: %s", plexbuff1, cb_get_strerror ());
+		/* do not output error if looking only for copybook deps */
+		if (!cb_flag_copybook_deps){
```

Comment:
How would one prevent the error of a not found include with GCC when generating the dependency file?

Reply:
I don't understand your question. You mean that we should add in the dependencies the dependencies that only appear in the generated C files ? FWIW, the `-fcopybook-deps` is a specific mode where we don't really care about missing COPY files, but other modes with `-M` will still complain about them missing.

Reply:
The question was: is there an option to let GCC generate a dependency file without complaining? If yes we should provide it the same way.

Reply:
I think it's `-MG`:       -MG In conjunction with an option such as -M requesting dependency
           generation, -MG assumes missing header files are generated files
           and adds them to the dependency list without raising an error.  The
           dependency filename is taken directly from the "#include" directive
           without prepending any path.  -MG also suppresses preprocessed
           output, as a missing header file renders this useless.

My implementation of -MG still complains about the error, but adds the file to the list of dependencies. I can modify my PR to remove the error in this case, but it will not be possible to suppress the output as `cobc` will probably have already started generating it when the error occurs.

Reply:
When this is in, then I'm quite sure we can suppress the error, too.

Reply:
Just pushed a version that does not complain with `-MG` about the error (see the last test in `used_binaries.at`)